src/Twig/AppExtension.php

```diff
@@ -28,4 +53,14 @@ public function getTextareaRows(string $text): int
     {
         return max(10, substr_count($text, "\n"));
     }
+
+    /**
+     * Get the name of the given language. This adds a few translations that don't exist in Intuition.
+     * @param string|null $lang
+     * @return string
+     */
+    public function getOcrLangName(?string $lang = null): string
```

Comment:
FYI, the original idea was to extend ToolforeBundle's `Extension` class and override the `getLangName` method. However we can't do this because ToolforgeBundle type-hints `Session` rather than `SessionInterface` which confuses Symfony in this context (we'd have to specify the DI in services.yml). I can still do that if we feel it's better.